src/main/java/step4/domain/Cars.java

```diff
@@ -0,0 +1,33 @@
+package step4.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+
+    List<Car> carList;
+
+    public Cars() {
+        this.carList = new ArrayList<>();
+    }
+
+    public List<Car> getCarList() {
+        return carList;
+    }
+
+    public void createdList(Data data) {
+        String[] names = data.spliteName(",");
```

Comment:
```suggestion
    public void createList(Data data) {
        String[] names = data.spliteName(",");
```

void 메서드 인것으로 봐서 엘레강트 오브젝트 관점에서 manipulator 로 보입니다.
동사로 네이밍하는 건 어떠신가요?